src/lando/dockerflow/decorators.py

```diff
@@ -0,0 +1,45 @@
+import time
+import logging

```

Comment:
I think it's because I do not add `dockerflow` to [LINT_PATHS here](https://github.com/mozilla-conduit/lando/blob/c17b6829e6851381390cf5ccaa3741320c7d4415/src/lando/settings.py#L188).

Reply:
The linter caught it after I added it.